app/assemblers/combine_authors_and_contributors.rb

```diff
@@ -4,8 +4,16 @@ class CombineAuthorsAndContributors
   initialize_with :authors, :contributors, limit: 3
 
   def call
-    users = authors.random.limit(limit).to_a
-    users += contributors.random.limit(limit - users.count).to_a
+    users = randomize(authors, limit)
+    users += randomize(contributors, limit - users.count)
     users
   end
+
+  private
+  def randomize(users, limit)
+    return users.random.limit(limit).to_a if users.respond_to?(:random)
```

Comment:
This allows randomizing arrays, not only active record collections.

Reply:
Are both approaches in this document method used?

Reply:
(It's a nice pivot!)

Reply:
The other approach is used in `approaches#show`.